### ISSUE TITLE
Decide on what to do with video media

### ISSUE THREAD

**Issue by jamietsao** [jamietsao/codepath-mapstagram#44]
We can either show it (not sure how difficult that is) or just filter videos out.

**Comment by kristen** [comment]
It looks like videos have the image urls that images have, so if we don't use a video player, than I don't think we have to filter out videos. Not sure if this is always true (I was just looking at the media API for type="video")

**Comment by jamietsao** [comment]
Yeah, that's true.  I think videos always have an image url so that is indeed an option.
